models/actions/artifact.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+// This artifact server is inspired by https://github.com/nektos/act/blob/master/pkg/artifacts/server.go.
+// It updates url setting and uses ObjectStore to handle artifacts persistence.
+
+package actions
+
+import (
+	"context"
+	"fmt"
+
+	"code.gitea.io/gitea/models/db"
+	"code.gitea.io/gitea/modules/timeutil"
+	"code.gitea.io/gitea/modules/util"
+)
+
+const (
+	// ArtifactUploadStatusPending is the status of an artifact upload that is pending
+	ArtifactUploadStatusPending = -1
+	// ArtifactUploadStatusConfirmed is the status of an artifact upload that is confirmed
+	ArtifactUploadStatusConfirmed = 1
```

Comment:
Why not
```suggestion
	// ArtifactUploadStatusPending is the status of an artifact upload that is pending
	ArtifactUploadStatusPending = 0
	// ArtifactUploadStatusConfirmed is the status of an artifact upload that is confirmed
	ArtifactUploadStatusConfirmed = 1
```

or

```suggestion
	// ArtifactUploadStatusPending is the status of an artifact upload that is pending
	ArtifactUploadStatusPending = 1
	// ArtifactUploadStatusConfirmed is the status of an artifact upload that is confirmed
	ArtifactUploadStatusConfirmed = 2
```

Reply:
I prefer 1 and 2. default value of int is 0 in go.